bin/get-api

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
I would also consider renaming this file to `get-api-key` so it better reflects what the script is doing.

Reply:
Changes made and pushed.